src/profile/forms/elements/EditButton.jsx

```diff
@@ -13,7 +13,8 @@ function EditButton({
 }) {
   return (
     <Button
-      className={classNames('btn-sm btn-link', className)}
+      variant="link"
+      className={classNames('btn-sm', className)}
```

Comment:
Button has a `size="sm"` prop: https://react-bootstrap.github.io/components/buttons/#sizes

Reply:
Oh, cool, I'll use that.